### ISSUE TITLE
Add a "Haven't Tried" option

### ISSUE THREAD

**Issue by MonkeyJamboree** [Goctionni/KinkList#4]
I'd really like to see an option (a checkbox, perhaps?) for flagging each item on the list as something you haven't tried... but make it in addition to the colored bubbles, so that it indicates how much you'd like to try something versus how much you have liked something you've tried.

**Comment by Goctionni** [comment]
I like this idea! I'm thinking making it like a cross-hatched or striped circle could be used here.

I need to think this through a bit though. I think "haven't tried" can be confusing in the sense that, for online-roleplaying purposes (where Kinklist is used most, I think) it could relate to either having tried it in role-play or in person.

**Comment by Lord-Kamina** [comment]
I was going to open an issue for something similar but I think most likely the "Default" should be "Haven't tried" and you should add a "Want to try" instead.

Incidentally, there are quite a few things missing (Needleplay come immediately to mind), I'll probably make a PR to add those at some point.

**Comment by Goctionni** [comment]
There's a lot of contexts in which the change regarding 'having tried/wanting to try' something makes a lot of sense. However, the original/majority userbase is one of online-roleplay (text based). In which it can be less relevant. If you want to create a change-PR, it'd probably be best if the PR were for a new file (with new format for filename).

As for missing kinks, unless you made the list at least ten or twenty times longer, some kinks would always be missing. In some communities a specific subset of kinks may be more prevalent. For those purposes I would again suggest creating a separate file rather than trying to create a single list that caters to everyone.

**Comment by Lord-Kamina** [comment]
True, there's probably always going to be things missing; on the other hand, I don't really see any downside to having more info. I've read some of your rebuttals in other issues (for example the one regarding colors) and some of those make a lot of sense but I really don't see why having more options and/or a "want to try" would be negative in any way for the online RP community (other than their current sheets would be updated but I'd wager most people who use this remake it on a regular basis and would most likely seamlessly transition to a new version were it to become available). Also; being the original target audience aside, I'm not sure you can really assert that it's the main audience today. 

The reason I chimed in re: the "Want to try" option is because: 1) I only just became aware of this file. 2) I sent it to a couple friends and for the most part they were confused about what the default was or what to enter for things they hadn't done but had an interest in and 3) For the same reason I wouldn't add a random color (based on what you outlined on that one issue) I thought it'd be best to consult with you on the best way to add the extra option but again, I don't see how having an extra option would be in any way detrimental to anyone.
